src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -517,14 +517,14 @@ protected override void ExecuteCore(DynamoModel dynamoModel)
             protected override void SerializeCore(XmlElement element)
             {
                 var helper = new XmlElementHelper(element);
-                helper.SetAttribute("XmlFilePath", XmlFilePath);
+                helper.SetAttribute("FilePath", FilePath);
```

Comment:
not sure we should touch any of this xml serialization as hopefully we won't be serializing any more xml.